views/partials/dashboards/preferences.hbs

```diff
@@ -6,13 +6,13 @@
       </div>
     </div>
     <div class="row jst-cntr">
-      <div class="col-8 card global-comm">
+      <div class="col-8 global-comm">
         <h2 class="section-headline">{{ getString "global-communication" }}</h2>
         <section>
           {{#each communicationOptions}}
             <label class="radio-container">
               <input class="radio-comm-option" data-comm-option="{{ optionId }}" data-form-action=" update-comm-option" type="radio" checked="checked" name="1">
-              <span class="radio-label">{{ labelString }}</span>
+              <span class="radio-label">{{{ labelString }}}</span>
```

Comment:
Can we do this without triple-stash syntax? I traced thru and see that `labelString` is currently only passed a `primaryEmail`, which is one of the `verifiedEmails` - so it shouldn't contain any malicious markup.

But I'm worried we might change the way this works later, and malicious markup might be able to sneak in.

Reply:
Yep - we need it to pass a span that bolds out the user's email address in that string.